CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## dbt 0.11.0 (unreleased, codename: Isaac Asimov)
+
+### Features
+
+- Extend catalog and manifest to also support Snowflake, BigQuery, and Redshift
```

Comment:
and postgres :)

Reply:
I'm pretty sure postgres was actually supported in Betsy Ross! I can call out postgres here too if you prefer though.